js/big-picture.js

```diff
@@ -0,0 +1,46 @@
+import { initCommentsList, renderComments } from './comment.js';
+
+const bigPictureElement = document.querySelector('.big-picture');
+const bodyElement = document.querySelector('body');
+const closePictureButton = bigPictureElement.querySelector('.big-picture__cancel');
+
+const hidePicture = () => {
+  bigPictureElement.classList.add('hidden');
+  bodyElement.classList.remove('modal-open');
+
+  document.removeEventListener('keydown', onDocumentKeydown);
+};
+
+function onDocumentKeydown(evt) {
+  if (evt.key === 'Escape') {
+    evt.preventDefault();
+    hidePicture();
+  }
+}
+
+const onClosePictureButton = () => {
+  hidePicture();
+};
+
+
+const renderPicture = ({ url, description, likes }) => {
+  bigPictureElement.querySelector('.big-picture__img img').src = url;
+  bigPictureElement.querySelector('.big-picture__img img').alt = description;
```

Comment:
тут два раза ищется один и тот же элемент, сама процедура поиска давольно ресурсоемкая, можно найти элемент один раз и далее просто его использовать

```suggestion
const bigPictureImgElement = bigPictureElement.querySelector('.big-picture__img img');
 bigPictureImgElement.src = url;
 bigPictureImgElement.alt = description;
```